dandi/download.py

```diff
@@ -853,12 +869,35 @@ def __exit__(
         exc_tb: TracebackType | None,
     ) -> None:
         assert self.fp is not None
+        if exc_type is not None or exc_val is not None or exc_tb is not None:
```

Comment:
`exc_tb` is not used in the log here.

Reply:
first I had it but then kicked out from actual log -- the point is that if any of them provided, exception happened.